x-pack/plugins/ml/public/application/components/annotations/annotations_table/annotations_table.js

```diff
@@ -644,15 +644,18 @@ export class AnnotationsTable extends Component {
         name: CURRENT_SERIES,
         dataType: 'boolean',
         width: '0px',
+        render: () => '',
       }
     );
+
+    const items = this.getAnnotationsWithExtraInfo(annotations);
     return (
       <Fragment>
         <EuiInMemoryTable
           error={searchError}
           className="eui-textOverflowWrap"
           compressed={true}
-          items={this.getAnnotationsWithExtraInfo(annotations)}
+          items={items}
           columns={columns}
```

Comment:
The `Label` column should sort numerically:

![image](https://user-images.githubusercontent.com/7405507/87790634-c9f19d00-c838-11ea-8a8e-4bd2663d17a6.png)

I think you can just replace `sortable: true` with something like `sortable: (key) => +key,`

Reply:
Updated the sort for numerical labels here https://github.com/elastic/kibana/pull/72204/commits/5e43657cd8cbd90698b1d74609e755d9e97161f8